llama2.mojo

```diff
@@ -586,7 +586,7 @@ fn softmax(inout x: TensorF32, start: Int, end: Int):
 
     @parameter
     fn _norm[_nelts: Int](ii: Int):
-        x.simd_store[_nelts](start + ii, x.simd_load[_nelts](start + ii) / ssum)
+        x.load[width=_nelts](start + ii, x.load[width=_nelts](start + ii) / ssum)
```

Comment:
`x.store` to replace first instance of `x.load`, since we are updating the value.